server/routerlicious/packages/routerlicious-base/src/alfred/routes/api/documents.ts

```diff
@@ -229,6 +229,25 @@ export function create(
 		},
 	);
 
+    /**
+     * Delete a document
+     */
+    router.post(
+        "/:tenantId/document/:id/delete",
+        validateRequestParams("tenantId", "id"),
+		verifyStorageToken(tenantManager, config, tokenManager),
```

Comment:
This should be verify delete token?

Reply:
I refactor validateTokenRevocationClaims() to validateTokenClaimsScopes(expectedScopes: string), to reuse the same code and check tokenRevocation claims and delete token claims, since the logic are the same.
One thing I noticed is that this generated test (packages/services-utils/src/test/types/validateServerServicesUtilsPrevious.generated.ts) complains about N/N-1 case at build time, because method name and signature changes. Any suggestions to fix this? @zhangxin511 @yunho-microsoft 
If there isn't a good way to address this N/N-1 test, I'll probably copy same logic of validateTokenRevocationClaims and create another function for delete token claims.

Reply:
@zhajie, you can also check with Gaurav to see he has any insights about the N/N-1 test.

Reply:
I think your PR missed this file: `validateServerRouterliciousBasePrevious.generated.ts`

This file is generated automaticlly and should included in the PR, the goal of it is trying to stop us making breaking changes unintentionally. :)

You can look the file at line 26 for the comment of this failing test, you should find something similar like 
![image](https://user-images.githubusercontent.com/3719246/236215227-1eb4901d-fc9b-4834-8957-08b3119d276e.png)

Here is an example:
![image](https://user-images.githubusercontent.com/3719246/236215515-bbebaf9b-e542-413d-9592-955e6f300091.png)

Details: https://github.com/microsoft/FluidFramework/pull/15264/files#diff-b02b1deb5db6142494fbc5c949e92d56151e4362eb5cbe3bb7b294fd394330e3

Please also add an entry in the breaking.Md under server/routerlicious folder

Reply:
Resolved the test failures as suggested. Thanks @zhangxin511